Docs/Követelmény_specifikáció.md

```diff
@@ -2,9 +2,9 @@
 
 ## 1. Áttekintés
 
-Ez a dokumentum egy modern és felhasználóbarát könyves webshop fejlesztéséhez készült követelmény specifikációt tartalmaz. Az online könyvkereskedelmi piac átalakulása miatt a hagyományos könyves webshopok gyakran elavultak és nem felelnek meg a mai vásárlói igényeknek. Az új webshop célja, hogy egy letisztult és átlátható platformot biztosítson a könyvrajongók számára, amely mind az új kiadványok, mind a ritka gyűjtői példányok és antik könyvek böngészését és vásárlását lehetővé teszi.
+Ez a dokumentum a könyv-webshopunk fejlesztéséhez készült követelményspecifikációt tartalmazza. Felméréseink szerint a felhasználók a hagyományos webshopokat elavultnak és nehezen használhatónak találják. Célunk, hogy egy letisztult, átlátható felületet biztosítsunk, és emellett antik és ritka kiadványokkal is kedvezzünk a könyvrajongóknak.
 
-A fejlesztés fő fókuszában a felhasználói élmény, az egyszerű navigáció és az átfogó keresési lehetőségek állnak, hogy a vásárlók könnyen megtalálják és megvásárolhassák az általuk keresett könyveket. A rendszer emellett közösségi funkciókat is kínál, mint például értékelési és véleményezési lehetőségek, amelyek elősegítik az olvasói közösség építését és a közösségi élményt. A webshop egy másodlagos piacot is biztosít, ahol az olvasók használt könyveiket is eladhatják vagy cserélhetik.
+Fejlesztésünk fő célja a kiváló felhasználói élmény, amit egyszerű navigációval és széles körű keresési lehetőségekkel kívánunk elérni. Fontos, hogy vásárlóink könnyen és gyorsan megtalálhassák, majd megvásárolhassák a keresett könyveket. A rendszer emellett közösségi funkciókat is kínál, például értékelési és véleményezési lehetőségeket, amelyek elősegítik az olvasói közösség építését és erősítik a közösségi élményt. A webshop továbbá egy másodlagos piacot is biztosít, ahol az olvasók használt könyveiket eladhatják vagy cserélhetik.
```

Comment:
"A webshop továbbá egy másodlagos piacot is biztosít, ahol az olvasók használt könyveiket eladhatják vagy cserélhetik." erre a doksi többi része nem utal